internal/provider/helper_test.go

```diff
@@ -1,7 +1,6 @@
-package gitlab
+package provider
```

Comment:
There are some helper functions in this file that I think go with your change of adding the `testGitlabClient` variable. Can we retire the `testAccNewClient` function, remove the `client *gitlab.Client` argument from the other functions, and remove the `client` field from `testAccGitlabProjectContext`?

Reply:
Thanks!